docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+
+This pattern also helps you to easily execute jobs in different environments without code changes. In this example, we dynamically define a repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+### For ops and graphs
+
+
+Although assets are useful in many ways, [ops](/concepts/ops-jobs-graphs/ops) and [graphs](/concepts/ops-jobs-graphs/graphs), are part of the Dagster ecosystem and can be integrated into your project structure.
+
+If you have use cases that don't meet the criteria for using assets, you may still use ops and graphs to build out these use cases. For example:
+
+- Sending an email or Slack message on a regular basis to users 
+- Reading a file of user IDs and change the value of a particular attribute for each user 
+- Scanning a data warehouse for tables that haven't been used in months and delete them
+
+In these cases, we recommend keeping all ops together in an 'ops_graphs/' directory. Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.  
+
+
+### For multiple repositories
+
+So far, we've discussed our recommendations for structuring a large project which contains only one repository. Dagster also allows you to structure a project with multiple Dagster repositories. We don't recommend over-abstracting too early, and in most cases, one repository should be sufficient. The pattern we found useful is to use multiple repositories is to keep conflicting dependencies separate, where each Dagster repository can keep their own package requirements (e.g., `setup.py`) and deployment specs (e.g., `Dockerfile`).
+
+### For other folders
+
+#### For tests
+
+Dagster believes that testing is a fundamental part of building a maintainable data application and have integrated CI/CD processes. We recommend setting up a separate folder structure that mirrors the main project, (e.g. having a folder for test assets with any applicable subfolders) which contains the unit tests for each of the components of the data pipeline. 
```

Comment:
🤔 Is the belief statement/first sentence here necessary? Same sentiment to my other comment about positioning, I think we should just focus on our recommendations and cut everything else as much as possible to make this short and sweet.